admins/pageflow/user.rb

```diff
@@ -1,22 +1,17 @@
 module Pageflow
   ActiveAdmin.register User do
-    menu :priority => 2
+    menu priority: 2, if: proc { authorized?(:index, current_user) }
```

Comment:
We could, but `@user.memberships.on_accounts.where(role: 'manager').any?` is simpler and more performant than
```
      manager_accounts = AccountPolicy::Scope
                         .new(@user, Account).member_addable
      managed_user_accounts = AccountPolicy::Scope
                              .new(@managed_user, Account).resolve
      (manager_accounts & managed_user_accounts).any?
```